package-lock.json

```diff
@@ -1085,7 +1085,8 @@
 		"asynckit": {
 			"version": "0.4.0",
 			"resolved": "https://registry.npmjs.org/asynckit/-/asynckit-0.4.0.tgz",
-			"integrity": "sha1-x57Zf380y48robyXkLzDZkdLS3k="
+			"integrity": "sha1-x57Zf380y48robyXkLzDZkdLS3k=",
+			"dev": true
```

Comment:
Do we need the development versions of these packages?

Reply:
That got auto updated when I did an npm install.

Reply:
Looks like these packages won't be installed if the `--production` flag is used. Not sure if this is what we want. See the last response here:
https://stackoverflow.com/questions/49809490/what-is-dev-true-in-package-lock-json-for

Do you mind testing the changes after removing these flags? Sorry, I am not well-versed in npm, trying to make sure this won't break production behavior.

Reply:
The dev=true is set because the npm packages have been added as dev dependencies. I noticed that the flag got auto-data when I added form-data as a dev dependency in package.json. I will look into your suggestion.

Reply:
Made changes that no longer require the form-data package to be installed.